### PR TITLE
Update procedure for removing an additional network

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1758295

### CODE REVIEW

File: modules/nw-multus-delete-network.adoc [hunk 1]
```diff
@@ -8,6 +8,13 @@ As a cluster administrator, you can remove an additional network from your
 {product-title} cluster. The additional network is not removed from any Pods it
 is attached to.
 
+[IMPORTANT]
+====
+In {product-title} 4.2.0 you must manually delete the additional network CR after removing it from the Cluster Network Operator configuration.
```
Comment (bobfuru) [inline modules/nw-multus-delete-network.adoc]: Are we certain this will apply only to this z-stream release (`4.2.0`), rather than just `4.2`?

**Review by bobfuru — COMMENTED** [review]
One question but LGTM otherwise!

File: modules/nw-multus-delete-network.adoc (hunk 1 above)
Comment (jboxman) [reply modules/nw-multus-delete-network.adoc]: @bobfuru, it's targeted to be fixed in 4.3.0 and back ported to 4.2.1.

**Review by weliang1 — COMMENTED** [review]
LGTM

**Comment by jboxman** [comment]
@weliang1 thank you!

**Pull request merged**